### PR TITLE
Es6 refactor

### PR DESCRIPTION
**Pull request by shellscape**
Fixed a few bugs. Got rid of Babel.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const del = require('del');
```
Comment (jwilsson) [inline gulpfile.js]: Looks like this is unused, right?

File: lib/node.js [hunk 2]
```diff
@@ -56,31 +58,39 @@ export default class Node {
...
+
     let cloned = this.clone(overrides);
 
     this.parent.insertAfter(this, cloned);
     return cloned;
   }
 
-  replaceWith (...nodes) {
+  replaceWith () {
+    let nodes = Array.prototype.slice.call(arguments);
```
Comment (jwilsson) [inline lib/node.js]: `Array.from` should work here, right?

http://node.green/#Array-static-methods

File: package.json [hunk 3]
```diff
@@ -1,11 +1,20 @@
 {
   "name": "postcss-values-parser",
-  "version": "0.1.8",
+  "version": "1.0.0",
```
Comment (jwilsson) [inline package.json]: Yay! 🎉

File: package.json [hunk 4]
```diff
@@ -18,48 +27,23 @@
...
-    "babel-core": "^6.4.0",
-    "babel-eslint": "^7.1.1",
-    "babel-plugin-add-module-exports": "^0.2.1",
-    "babel-plugin-precompile-charcodes": "^1.0.0",
-    "babel-preset-es2015": "^6.3.13",
-    "babel-preset-es2015-loose": "^8.0.0",
-    "babel-preset-stage-0": "^6.3.13",
     "chai": "3.5.0",
     "chai-shallow-deep-equal": "^1.4.0",
     "del": "^2.2.0",
```
Comment (jwilsson) [inline package.json]: Just a friendly reminder to delete it here as well.

File: test/word.js [hunk 5]
```diff
@@ -51,6 +53,20 @@ describe('Parser → Word', () => {
...
+    },
+    {
+      it: 'should parse custom variables',
+      test: '--color',
+      expected: [
+        { type: 'word', value: '--color' }
+      ]
+    },
+    {
+      it: 'should parse custom variables',
```
Comment (jwilsson) [inline test/word.js]: Copy/paste error from above?

**Comment by jwilsson** [comment]
Something to think about: I've been thinking of publishing a `eslint-config-lesshint` but we disagree on some things (brace placement, spacing, etc.) 😉 . But that's always easy to override.

File: lib/parser.js [hunk 6]
```diff
@@ -331,14 +337,13 @@ export default class Parser {
...
 
     indices.forEach((ind, i) => {
-      let index = indices[i + 1] || word.length;
-      let value = word.slice(ind, index);
-      let node;
+      let index = indices[i + 1] || word.length,
+        value = word.slice(ind, index),
+        node;
 
       if (~hasAt.indexOf(ind)) {
```
Comment (jwilsson) [inline lib/parser.js]: Could `hasAt.includes` be used instead?

http://node.green/#String-prototype-methods

File: lib/node.js (hunk 2 above)
Comment (shellscape) [inline lib/node.js]: 6 of one, half dozen of another :)

File: lib/parser.js (hunk 6 above)
Comment (shellscape) [inline lib/parser.js]: is there a really good reason for `includes` over `indexOf`?

Comment (jwilsson) [inline lib/parser.js]: Bit easier to read IMO, but just a thought.

**Comment by shellscape** [comment]
re: the eslint config, yeah I'm planning on a shared config for all my projects as well. it's on my todo list. also need to get one going for the company I work for.

File: gulpfile.js (hunk 1 above)
Comment (shellscape) [inline gulpfile.js]: :squid: (closest thing to squashed i could find)

File: package.json (hunk 4 above)
Comment (shellscape) [inline package.json]: :squid:

File: test/word.js (hunk 5 above)
Comment (shellscape) [inline test/word.js]: :squid:

**Pull request merged**